src/Api/ApiResourceRegistrar.php

```diff
@@ -54,8 +54,9 @@ public function __construct(Router $api)
      * @param string $resourceName URI of resource
      * @param string $controller FQDN Class name of Controller, which contains action method
      * @param array $options options, passed to router on route registration
+     * @param string $modelClass Model class short name to bind in controller method instead id.
```

Comment:
what mean 'Short' name? Can't we use SomeModel::class? If not, isn't better to use Str::singular($resourceName) (or same for last segment if resource is some URL path)

Reply:
or on the contrary, can't we use model class name to retrieve resource name?